cmd/vsphere-xcopy-volume-populator/README.md

```
@@ -136,7 +136,37 @@ spec:
 
 # Secret with storage provider credentials
 
-## Hitachi Vantara
+The secret with storage provider credentials must contain the following environment variables:
+
+## vSphere Provider Settings
+| Key | Required/Optional | Description |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)